### PR TITLE
Merge dev into Master for v1.1.0

### PR DESCRIPTION
**Pull request by element15**
The current state of the `dev` branch appears to be stable, and has fixed many of the bugs and inaccuracies present int v1.0.0.

### CODE REVIEW

File: generate_leaf_chart.m [hunk 1]
```diff
@@ -0,0 +1,33 @@
+% Seperate from the main functionality of the program, this script
+% generates a chart for the flow characterization of the test sprinkler
+
+clear;
+[a, r2] = sprinkler_1_data();
+flow_func = @(h) a * h.^0.5;
+parameters;
+
+p_orig = (5:2.5:35); % Pressure, psi
+v_dot_orig = [6.83, 7.77, 8.75, 9.53, 10.30, 10.82, 11.87, 12.29, ...
```
Comment (rakmo97) [inline generate_leaf_chart.m:10]: Where do these numbers come from in v_dot_orig?

**Comment by rakmo97** [comment]
@element15  I don't have any real comments.   Should I merge the pull request or let Emily take a look first?

File: generate_leaf_chart.m (hunk 1 above)
Comment (element15) [reply generate_leaf_chart.m:10]: Those are from me buying an impulse sprinkler similar to the ones to be used in Quesimpuco and measuring the volumetric throughput at various pressure levels.

**Comment by element15** [comment]
> @element15 I don't have any real comments. Should I merge the pull request or let Emily take a look first?

I'll message her and see if she's available today or tomorrow. If not it should be find to merge.

**Review by rakmo97 — APPROVED** [review]

**Pull request merged**